app/server.go

```diff
@@ -670,6 +672,12 @@ func NewServer(options ...Option) (*Server, error) {
 
 	s.initPostMetadata()
 
+	docExtractorService, err := docextractor.NewLocalExtractorService(*s.Config().FileSettings.ArchiveRecursion)
```

Comment:
Here we need to better decide when to use one or another based on the configuration, now we only use the local spawn process, that was the most interesting one for this case, the other to are just subcases of this one.